include/z64speed_meter.h

```diff
@@ -7,12 +7,24 @@ struct GameState;
 struct GraphicsContext;
 
 typedef struct SpeedMeter {
-    /* 0x00 */ char unk_00[0x20];
+    /* 0x00 */ char unk_00[0x18];
+    /* 0x18 */ s32 x; // Unused
+    /* 0x1C */ s32 y;
 } SpeedMeter; // size = 0x20
 
 void SpeedMeter_Init(SpeedMeter* this);
 void SpeedMeter_Destroy(SpeedMeter* this);
 void SpeedMeter_DrawTimeEntries(SpeedMeter* this, struct GraphicsContext* gfxCtx);
-void SpeedMeter_DrawAllocEntries(SpeedMeter* this, struct GraphicsContext* gfxCtx, struct GameState* gameState);
+void SpeedMeter_DrawAllocEntries(SpeedMeter* this, struct GraphicsContext* gfxCtx, struct GameState* state);
+
+extern volatile OSTime gRSPAudioTimeTotal;
+extern volatile OSTime gRSPGfxTimeTotal;
+extern volatile OSTime gRDPTimeTotal;
+extern volatile OSTime gGraphUpdatePeriod;
+extern volatile OSTime gRSPAudioTimeAcc;
+extern volatile OSTime gRSPGfxTimeAcc;
+extern volatile OSTime gRSPOtherTimeAcc;
+extern volatile OSTime D_801FBB18;
```

Comment:
Is this unused? It seems very obvious as to it's use by surrounding variables, but would we still want to name this `sPad`?

Reply:
The only issue is I would very much prefer externing it in the header so I don't have to deal with bss as much. Perhaps we could name it something like `gUnkTimeAcc` to give it a unique name and can extern it without issue.